docs/design_proposals/sync-absolute-paths.md

```diff
@@ -0,0 +1,38 @@
+# Sync support absolute paths
```

Comment:
We can remove this design proposal since you're implementing the proposal and it is refining rather than changing Skaffold's default behaviours.

But we should change the documentation.  Here's my proposal:
```diff
diff --git docs/content/en/docs/pipeline-stages/filesync.md docs/content/en/docs/pipeline-stages/filesync.md
index a4bed9be3..2a71b8f66 100644
--- docs/content/en/docs/pipeline-stages/filesync.md
+++ docs/content/en/docs/pipeline-stages/filesync.md
@@ -26,8 +26,11 @@ Multiple types of sync are supported by Skaffold:
 ### Manual sync mode
 
 A manual sync rule must specify the `src` and `dest` field.
-The `src` field is a glob pattern to match files relative to the artifact _context_ directory, which may contain `**` to match nested files.
-The `dest` field is the absolute or relative destination path in the container.
+
+- The `src` field is a glob pattern to match files, which may contain `**` to match nested files.
+  Relative paths are resolved from the artifact _context_ directory.  Absolute paths may also be used.
+- The `dest` field is the absolute or relative destination path in the container.
+
```

Reply:
mark resolved